gui/src/pages/NewDockerNetwork.svelte

```diff
@@ -1,16 +1,16 @@
 <script lang="ts">
+  import CodeBlock from '../components/CodeBlock.svelte';
   import NewDockerComponent from './NewDockerComponent.svelte';
   import type { Params } from './NewDockerComponent.svelte';
 
   export let params: Params;
 
-  const specExample = `# See https://github.com/compose-spec/compose-spec/blob/master/spec.md#Networks-top-level-element
-name: mynetwork`;
+  const code = `name: mynetwork`;
+  const href =
```

Comment:
nit: a better name for this would be something like `docsHref` or `docsUrl` or like that, since absent any qualifiers, it isn't clear what the href would mean.